debian/rules

```diff
@@ -218,11 +218,28 @@ override_dh_auto_install:
 		cp -a $(CURDIR)/debian/tmp/lib/$(DEB_HOST_MULTIARCH)/$$lib* $(TEMPLIBDIR); \
 	done
 	cp -a $(CURDIR)/debian/tmp/usr/lib/systemd/libsystemd-shared* $(TEMPLIBDIR)
-
-	set -e; \
-	for f in "/lib/$(DEB_HOST_MULTIARCH)/libnss_files.so.* /lib/$(DEB_HOST_MULTIARCH)/libnss_compat.so.* /lib/$(DEB_HOST_MULTIARCH)/libgcc_s.so.1 /sbin/e2fsck /sbin/fsck.vfat /sbin/fsck /bin/umount /bin/mount /bin/kmod /usr/bin/unsquashfs /sbin/dmsetup /usr/lib/snapd/snap-bootstrap /usr/lib/snapd/info /lib/systemd/systemd-bootchart /lib/systemd/system/snapd.recovery-chooser-trigger.service"; do \
-		LD_PRELOAD= LD_LIBRARY_PATH=$(TEMPLIBDIR) \
-			/usr/lib/dracut/dracut-install -D $(CURDIR)/debian/tmp --ldd $$f; \
+	set -e;									\
+	for f in								\
+		/lib/$(DEB_HOST_MULTIARCH)/libnss_files.so.*			\
+		/lib/$(DEB_HOST_MULTIARCH)/libnss_compat.so.*			\
+		/lib/$(DEB_HOST_MULTIARCH)/libgcc_s.so.1			\
+		/sbin/e2fsck							\
+		/sbin/fsck.vfat							\
+		/sbin/fsck /bin/umount						\
+		/bin/mount							\
+		/bin/kmod							\
+		/usr/sbin/modprobe						\
```

Comment:
Is this preserving modprobe as a symlink or does it make a copy of kmod? Also, I think this is a good opportunity to add the rest of possible symlinks to kmod: lsmod, rmmod, insmod, modinfo, depmod. It is always annoying having to create them by hand when debugging the initramfs.

Reply:
`dracut-install` preserves the symlinks. I have added the missing ones.